### ISSUE TITLE
Training dataset

### ISSUE THREAD

**Issue by hugoabonizio** [todd-cook/ML-You-Can-Use#4]
Hi @todd-cook!

I'm attempting to reproduce your results using BERT on the Crowdflower Search Results Relevance dataset, but I'm facing some issues. [Your notebook](https://github.com/todd-cook/ML-You-Can-Use/blob/master/searching/search_results_relevance_using_BERT.ipynb) says that the training set contains 20,571 labeled samples, but when I download the dataset from [Kaggle](https://www.kaggle.com/c/crowdflower-search-relevance/data) it contains only 10,159 samples.

Am I missing something?

**Comment by todd-cook** [comment]
Hi Hugo,

Good catch! Thank you for letting me know! I have just added the dataset
file to the repo.

https://github.com/todd-cook/ML-You-Can-Use/blob/master/searching/kaggle_ecommerce_complete_Rank_Crowd_AGG_Descriptions_ALL_ROWS_utf8.csv

As I recall, it doesn't make a lot of difference but it helps a little and
it will certainly help you reproduce the results.
The source of the dataset is the sponsor of the project, Lukas Biewald and
Crowdflower; somewhere in the Kaggle discussion forum he mentioned that the
updated data set was made available and I grabbed that.

Let me know if you have any other problems or questions.
Enjoy your journey!

Todd
